### PR TITLE
add 4.4.0.Final and 10.0.0.GA to the website

### PR DESCRIPTION
**Pull request by nickboldt**
add 4.4.0.Final and 10.0.0.GA to the website

### CODE REVIEW

File: _config/products.yml [hunk 1]
```diff
@@ -789,12 +820,32 @@ jbt_core:
             url: https://repository.jboss.org/nexus/service/local/artifact/maven/redirect?r=snapshots&g=org.jboss.tools.browsersim&a=org.jboss.tools.browsersim-standalone&v=3.8.0-SNAPSHOT&e=zip
             #url: http://download.jboss.org/jbosstools/neon/snapshots/builds/jbosstools-browsersim-standalone_master/latest/jbosstools-4.4.0.Alpha1-browsersim-standalone.zip
             #md5_url: http://download.jboss.org/jbosstools/neon/snapshots/builds/jbosstools-browsersim-standalone_master/latest/jbosstools-4.4.0.Alpha1-browsersim-standalone.zip.sha256
+      4.4.0.Final:
+        update_site_url: http://download.jboss.org/jbosstools/neon/development/updates/
```
Comment (maxandersen) [inline _config/products.yml]: why is this development and not stable ?

File: _config/products.yml [hunk 2]
```diff
@@ -789,12 +820,32 @@ jbt_core:
...
+        marketplace_url: https://marketplace.eclipse.org/node/1617241
+        marketplace_install_url: http://marketplace.eclipse.org/marketplace-client-intro?mpc_install=1617241
+        release_date: 2016-05-27
+        supported_jbt_is_version: 4.3.0.Final
+        blog_announcement_url: /blog/ga-for-neon.html
+        documentation_url: http://docs.jboss.org/tools/
+        zips:
+          - name: Update site (including sources) bundle of all JBoss Core Tools
+            file_size: 456MB
+            url: http://download.jboss.org/jbosstools/static/neon/development/updates/core/jbosstools-4.4.0.Final-updatesite-core.zip
```
Comment (maxandersen) [inline _config/products.yml]: same here - should be stable not development.

**Comment by nickboldt** [comment]
s/development/stable/g for 4.4.0 & 10.0.0.

Thanks, Max. Good catch.

**Comment by alexeykazakov** [comment]
Pushed to master and production

**Pull request closed without merging**